feature/platform/transceiver/zr_tunable_parameters_test/README.md

```diff
@@ -0,0 +1,137 @@
+
+## Summary
+
+Validate setting 400ZR tunable parameters channel frequency and output TX
+launch power and verify corresponding telemetry values.
+
+### Goals
+* Verify full C band frequency tunability for 100GHz line system grid.
+* Verify full C band frequency tunability for 75GHz line system grid.
+* Verify adjustable range of transmit output power across -13 to -9 dBm
+  in steps of 1 dB.  
+
+
+## TRANSCEIVER-5.1
+
+*   Connect two ZR interfaces using a duplex LC fiber jumper such that TX
+    output power of one is the RX input power of the other module. Connection
+    between the modules should pass through an optical switch that can be
+    controlled through automation to simulate a fiber cut.  
+*   To establish a point to point ZR link ensure the following:
+      * Both transceivers states are enabled.
+      * Validate setting 400ZR optics module tunable laser center frequency
+        across frequency range 196.100 - 191.400 THz for 100GHz grid.
+      * Validate setting 400ZR optics module tunable laser center frequency
+        across frequency range 196.100 - 191.375 THz for 75GHz grid.  
+      * Specific frequency details can be found in 400ZR implementation
+        agreement under sections 15.1 ad 15.2 Operating frequency channel
+        definitions.
+      * Validate adjustable range of transmit output power across -13 to -9 dBm
+        range in steps of 1dB. So the module’s output power will be set to -13,
+        -12, -11, -10, -9 dBm in each step.
+
+*   With the ZR link established as explained above, for each configured
+    frequency and TX output power value verify that the following ZR
+    transceiver telemetry paths exist and are streamed for both the ZR optics.
+    * Frequency
+        *   /components/component/optical-channel/state/frequency
+        *   /components/component/optical-channel/state/carrier-frequency-offset/instant
+        *   /components/component/optical-channel/state/carrier-frequency-offset/avg
+        *   /components/component/optical-channel/state/carrier-frequency-offset/min
+        *   /components/component/optical-channel/state/carrier-frequency-offset/max
+    * TX Output Power
+        *   /components/component/optical-channel/state/output-power/instant
+        *   /components/component/optical-channel/state/output-power/avg
+        *   /components/component/optical-channel/state/output-power/min
+        *   /components/component/optical-channel/state/output-power/max
+
+* With above streamed data verify
+    * For each center frequency, laser frequency offset should not be more than
+      +/- 1.8 GHz max.
+    * When set to a specific target output power, transmit power control
+      absolute accuracy should be within +/- 1 dBm of the target value.
+    * For reported data check for validity: min <= avg/instant <= max
+
+   
+## TRANSCEIVER-5.2
+
+*   When the modules or the devices are still in a boot stage, they must not
```

Comment:
Is there another test which performs a reboot of a transceiver while streaming already?    

In any case, can you state the procedure you'd like to be used for this?   Something like:

* Start the  streaming defined in TRANSCEIVER 5.1
* Reboot the transceiver
* <suggest pass and fail criteria.  Clearly 'nil' is fail.  But it's not clear to me what the streamed values should be durning the reboot in order to pass? >